spec/fixtures/files/mail_with_invalid_to.eml

```diff
@@ -0,0 +1,29 @@
+X-Original-To: bd84c730a1ac7833e4d27253804516f7@reply.chatwoot.com
+Received: from mail.planetmars.com (mxd [192.168.1.1]) by mx.sendgrid.net with ESMTP id AAAA-bCCCCCC5DeeeFFgg for <bd84c730a1ac7833e4d27253804516f7@reply.chatwoot.com>; Sun, 31 Dec 2023 22:32:23.586 +0000 (UTC)
+From: "Mark Whatney" <>
+To: <vishnu@chatwoot.com>vishnu@chatwoot.com
```

Comment:
This is not an invalid email address right?

Reply:
@pranavrajs Even though it appears legit, it's invalid according to the RFC specs. This is from a failed email from our prod with some details redacted.

Reply:
@pranavrajs I am assuming this address it's invalid because the email appears twice, and the implementation does not allow it